docs/components/databases.rst

```diff
@@ -104,3 +104,23 @@ How to use the :code:`mysql` or :code:`mariadb` format?
 .. note::
     If the environment variables are not found, you will be asked to prompt the 
     information.
+
+Known limitations with MySQL and MariaDB
+----------------------------------------
+
+Do to a default setting that prevent users from creating stored functions that 
+will cause unsafe events to be written into the binary log, you will need to 
+contact your database manger (maybe your self) and ask them to
```

Comment:
It a little heavy to read. and to much back and forth. I suggestion a second rewrite, to soften the reading and enhancing the understanding.

Good work @AnonymousPoster

Reply:
I agree. I'll take a second review and see what I can do :smiling_imp:

Reply:
I've some minor changes that at first eyesight looks ~ok

https://github.com/funilrys/PyFunceble/pull/81